docs/databases/redis/deploy-redis-on-ubuntu-14-04.md

```diff
@@ -56,6 +61,33 @@ The Redis package in the Ubuntu 14.04 repository is outdated and lacks several s
         sudo apt-get update
         sudo apt-get install redis-server
 
+### Debian
+
+[Dotdeb](https://www.dotdeb.org) is a popular third party repository for Debian users looking for newer versions of the LAMP stack and it's related softwares than Debian provides.
+
+1.  Review the [list of mirrors](https://www.dotdeb.org/mirrors/) Dotdeb provides and select the one closest to your Linode.
+
+2.  Create the file `/etc/apt/sources.list.d/dotdeb.list` and copy the appropriate mirror information to it:
+
+    {: .file}
+    /etc/apt/sources.list.d/dotdeb.list
+    :   ~~~
+        deb http://ftp.utexas.edu/dotdeb/ stable all
+        deb-src http://ftp.utexas.edu/dotdeb/ stable all
+        ~~~
+
+3.  Download and install the GPG key, as documented [here](https://www.dotdeb.org/instructions/):
+
+        wget https://www.dotdeb.org/dotdeb.gpg
+        sudo apt-key add dotdeb.gpg
+
+4.  Update packages and install `redis-server` package:
+
+        sudo apt-get update
+        sudo apt-get install redis-server
+
+### Verify the Installation
+
 3.  Verify Redis is up by running `redis-cli`:
```

Comment:
@EdwardAngert strange, in diff it shows
`3.  Verify Redis is up by running `redis-cli`:`
but when I view the whole file, its

`1.  Verify Redis is up by running redis-cli:`

So maybe its not a problem. Anyway, I think it's a minor thing and could be fixed later.